docs/index.rst

```diff
@@ -33,9 +33,11 @@ You can always find the latest release `here <https://github.com/eth-cscs/refram
 Publications
 ============
 
-* *ReFrame: A regression framework for checking the health of large HPC
-  systems* [`slides <_static/files/reframe-cug17-slides.pdf>`__]
+* *ReFrame: A regression framework for checking the health of large HPC systems* [`slides <_static/files/reframe-cug17-slides.pdf>`__]
 
+* *ReFrame: A Regression Testing Tool for HPC Systems* [`slides <_static/files/reframe-bof-sc17-slides.pdf>`__]
+
+* *ReFrame: A Regression Testing Framework Enabling Continuous Integration of Large HPC Systems* [`slides <_static/files/reframe-hpac18-slides.pdf>`__]
```

Comment:
I wouldn't put the title in the listing, since they are all very similar. Instead I would suggest the following:

```rst
* `Presentation & Demo <_static/files/reframe-hpac18-slides.pdf>`__ @ `HPC Advisory Council 2018 <http://www.hpcadvisorycouncil.com/index.php>`__.
```

- Ideally, I would have a separate link for the presentation and demo (my example above doesn't do, cos I didn't have the link for the demo).
- Remember to put links to the venues, as I did in my example here.
- Also order the list from the newest to the oldest publication.